pytest.ini

```diff
@@ -0,0 +1,9 @@
+[pytest]
+markers =
+    run_this: testing marker for run this test, useful for development
+    deployment: marker for deployment
```

Comment:
I think that all deployment should be kept in fixtures. It shouldn't be done by tests.

Reply:
@fbalak I am just moving stuff from what we had done by previous runner to be able to run it by pytest to let us move to pytest runner as soon as possible. We still didn't decided how to handle deployment or cleanup here in pytest. This will be part of different PR if we decide to do it different way.

Reply:
I think, that since we will have to test also the installation itself, it is IMHO test.

Reply:
Ok